action.yml

```diff
@@ -226,12 +233,22 @@ runs:
         inputs.pull-request-change-detection
       }}')
       pull_request_branch = '${{ github.event.pull_request.base.ref || '' }}'
+      coverage = '${{ inputs.coverage }}'
+
+      # Validate GHA inputs
+      if coverage not in ('always', 'never', 'auto'):
+          print(
+              '::error ::`coverage` must have one of the values `always`,'
```

Comment:
In the future, it might be interesting to make use of the job summaries too. But let's count this as a possible improvement and keep it as is for now.